### PR TITLE
Minor Redux User change 

### PR DESCRIPTION
**Pull request by markus-leben**

### CODE REVIEW

File: src/App.js
Comment (JakeSifo) [inline src/App.js:1]: Was dis?

**Comment by masonisblue** [comment]
thanks for making the code a bit more dry. 

I'm sure many of us need to add properties to the userSlice, hopefully there won't be any breaking changes. For me I definitely need to add a bookmarks property still

**Review by masonisblue — APPROVED** [review]
thanks for making the code a bit more dry.

I'm sure many of us need to add properties to the userSlice, hopefully there won't be any breaking changes. For me I definitely need to add a bookmarks property still

File: src/server/firestore.js
Comment (JakeSifo) [inline src/server/firestore.js:1]: const firebaseConfig = {
  apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
  authDomain: process.env.REACT_APP_FIREBASE_AUTH_DOMAIN,
  projectId: process.env.REACT_APP_FIREBASE_PROJECT_ID,
  storageBucket: process.env.REACT_APP_FIREBASE_STORAGE_BUCKET,
  messagingSenderId: process.env.REACT_APP_FIREBASE_MESSAGING_SENDER_ID,
  appId: process.env.REACT_APP_FIREBASE_APP_ID,
  measurementId: process.env.REACT_APP_FIREBASE_MEASUREMENT_ID
};

**Review by JakeSifo — APPROVED** [review]

**Pull request merged**